Formula/certgraph.rb

```diff
@@ -0,0 +1,26 @@
+class Certgraph < Formula
+  desc "Intelligence tool to crawl the graph of certificate Alternate Names"
+  homepage "https://lanrat.github.io/certgraph"
+  url "https://github.com/lanrat/certgraph.git",
+    :tag => "20180217",
```

Comment:
4 space indent rather than 2 space on both `:tag` and `:revision` (but nowhere else).